# src/utils/exceptions.py
class APIException(Exception):
    """Exception personnalisée pour les erreurs liées aux API."""
    pass
